code/web/src/modules/user/Subscriptions.js

```diff
@@ -25,7 +28,8 @@ class Subscriptions extends PureComponent {
 
   // Runs on client only
   componentDidMount() {
-    this.props.getListByUser()
+    this.
+    props.getListByUser()
```

Comment:
any reason why this is on the next line?

Reply:
Nope! Nice catch!